python/ray/_private/worker.py

```diff
@@ -2708,14 +2708,14 @@ def put(
 @PublicAPI
 @client_mode_hook
 def wait(
-    ray_waitables: List[Union["ObjectRef[R]", "ObjectRefGenerator[R]"]],
+    ray_waitables: List[Union[ObjectRef, ObjectRefGenerator]],
```

Comment:
what is `R` and why we remove it?

Reply:
This is [type parameter syntax](https://docs.python.org/3/reference/compound_stmts.html#type-params). There's information [here](https://docs.python.org/3/library/typing.html#generics) on how it works, but this is a way of parameterizing a type using a generic. These forward references won't resolve unless the types are defined or imported here, so that's why they're changed to concrete references.